Api/Model/OrderDoesNotExistException.php

```diff
@@ -5,12 +5,21 @@
 
 namespace Auctane\Api\Model;
 
-use Exception;
+use Magento\Framework\Exception\LocalizedException;
 
-class OrderDoesNotExistException extends Exception
+
+/**
+ * Class OrderDoesNotExistException
+ * @package Auctane\Api\Model
+ */
+class OrderDoesNotExistException extends LocalizedException
 {
+    /**
+     * OrderDoesNotExistException constructor.
+     * @param $orderId
+     */
     public function __construct($orderId)
     {
-        parent::__construct("Order '{$orderId}' does not exist", 400);
+        parent::__construct(__("Order '%1' does not exist", $orderId), null, 400);
```

Comment:
If 400 is the HTTP status code, could consider using the const that shows up elsewhere for consistency (HTTP_BAD_REQUEST)